### DISCUSSION TITLE
Arlec Christmas Tree Lights

### DISCUSSION THREAD

**Discussion by jmr000** [milo526/homebridge-tuya-web#362] [Q&A]
Arlec Christmas Tree Lights

Hi All, trying to expose Arlec Christmas tree lights to HomeKit via homebridge but am not having much luck. 

Using smart life app > virtual ID > Tuya web platform plugin > expose as light.

I have other lights running fine through smart life but it seems as the Christmas lights have more options in the app it doesn’t communicate.

Any ideas of how to control them.
Just looking for on/off & rgb control.

**Comment by milo526** [comment]
Please post the result of [the steps under "How to check whether the api this library uses can control your device".](https://github.com/milo526/homebridge-tuya-web#how-to-check-whether-the-api-this-library-uses-can-control-your-device)

Without this it is hard for me to help you with regards to this specific issue.

**Comment by jmr000** [comment]
Ran the code and it doesn’t appear in the output. 
Thanks for your assistance I will reach out to Tuya support. 

Jacob Richter
